### PR TITLE
Example/blog previous next navigation

### PR DESCRIPTION
**Pull request by spences10**
resubmit with changes suggested by @tannerlinsley 

comments https://github.com/nozzle/react-static/pull/682#issuecomment-424392120

### CODE REVIEW

File: examples/previous-next-navigation/src/App.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Router, Link } from 'react-static'
+import { hot } from 'react-hot-loader'
+//
+import Routes from 'react-static-routes'
```
[the remark this answers is not in the document]
Comment (spences10) [reply examples/previous-next-navigation/src/App.js]: Like this?

```js
import { Router, Link, Routes } from 'react-static'
```
??

File: examples/previous-next-navigation/src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { withSiteData } from 'react-static'
+//
+import logoImg from '../logo.png'
```
[the remark this answers is not in the document]
Comment (spences10) [reply examples/previous-next-navigation/src/pages/index.js]: done!

File: examples/previous-next-navigation/src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (spences10) [reply examples/previous-next-navigation/src/App.js]: I've done that

[the remark this answers is not in the document]
Comment (tannerlinsley) [reply examples/previous-next-navigation/src/App.js]: Yep!

**Comment by tannerlinsley** [comment]
Perfect!!

**Pull request merged**